drivers/logserial.py

```diff
@@ -21,50 +21,38 @@
 
 
 class LogSerial(Thread):
-    def __init__(self, config, bus, com=None):
+    def __init__(self, config, bus):
         Thread.__init__(self)
         self.setDaemon(True)
         self.should_run = Event()
         self.should_run.set()
 
-        if com is None:
-            if 'port' in config:
-                self.com = serial.Serial(config['port'], config['speed'])
-                self.com.timeout = 0.01  # expects updates < 100Hz
-            else:
-                self.com = None
+        self.output_thread = Thread(target=self.run_output)
+
+        if 'port' in config:
+            self.com = serial.Serial(config['port'], config['speed'])
+            self.com.timeout = 0.01  # expects updates < 100Hz
```

Comment:
Should `timeout` be part of `config` as well?

Reply:
yes, but maybe later. In "must have" TODO list reset for CAN bridge, which I would add in config.